src/SDKs/CognitiveServices/dataPlane/Vision/ComputerVision/ComputerVision/Generated/Models/ImageType.cs

```diff
@@ -50,13 +50,13 @@ public ImageType(double clipArtType = default(double), double lineDrawingType =
         /// Gets or sets confidence level that the image is a clip art.
         /// </summary>
         [JsonProperty(PropertyName = "clipArtType")]
-        public double ClipArtType { get; set; }
+        public int ClipArtType { get; set; }
```

Comment:
Was this already an incorrect SDK which is being fixed or has something at the service end changed? Applies to other parameters/members too

Reply:
The service hasn't changed, we were preparing to make a version bump due to the name change (ComputerVisionAPI->ComputerVisionClient) so we're piling on with other changes we'd intended to fix.  The service has not changed in a while.